src/bytes.test.ts

```diff
@@ -458,6 +460,7 @@ describe('createDataView', () => {
 
   describe('Node.js Buffer', () => {
     it('returns a DataView from a byte array', () => {
+      // eslint-disable-next-line no-restricted-globals
```

Comment:
Hmm. Perhaps we should extend the node ESLint config for tests as well? They are run directly with node, and that would bring in these globals automatically as well.

Reply:
Good point! I've updated the ESLint config and removed these lines.